war/src/main/less/abstracts/colors.less

```
@@ -1,84 +1,151 @@
+
+  // Header classic colors
+  --header-bg-classic: #000;
+  --header-link-bg-classic-hover: #404040;
+  --header-link-bg-classic-active: #404040;
+
+  // Header new UI colors
+  --header-bg-v2: var(--primary);
+  --brand-link-color-hover-v2: var(--primary-hover);
+  --brand-link-color-active-v2: var(--primary-active);
+  --header-link-bg-hover-v2: var(--primary-hover);
+  --header-link-bg-active-v2: var(--primary-active);
+
+  // Header dimensions
+  --header-item-border-radius: 4px;
+
+  // Breadcrumbs and footer
+  --breadcrumbs-bg: #f8f8f8;
+  --breadcrumbs-border: #eaeff2;
+  --breadcrumbs-text-color: #4d545d;
+
+  // Monitor / bell-alert new colors
+  --monitor-bg-v2: #f8d7da;
+  --monitor-color-v2: #721c24;
+
+  // Alert banners
+  // Default
+  --alert-default-icon-color: #2196f3;
+  --alert-default-bg-color: #d1ecf1;
+  --alert-default-border-color: #bee5eb;
+  --alert-default-color: #0c5464;
+
+  // Success
+  --alert-success-icon-color: #4caf50;
+  --alert-success-bg-color: #d4edda;
+  --alert-success-border-color: #c3e6cb;
+  --alert-success-color: var(--success);
+
+  // Warning
+  --alert-warn-icon-color: #ff9800;
+  --alert-warn-bg-color: #fff3cd;
+  --alert-warn-border-color: #ffeeba;
+  --alert-warn-color: #856404;
+
+  // Error
+  --alert-err-icon-color: #f44336;
+  --alert-err-bg-color: #f8d7da;
+  --alert-err-border-color: #f5c6cb;
+  --alert-err-color: #721c24;
+
+  // Typography
+  --text-color: var(--black);
+  --text-color-grey-light: #747c82;
+  --text-inactive: var(--text-color-grey-light);
+
+  // Buttons
+  --btn-primary-bg: var(--primary-hover);
+  --btn-primary-bg-hover: var(--primary);
+  --btn-primary-bg-active: var(--primary-active);
+  --btn-secondary-color: var(--secondary);
+  --btn-secondary-bg: var(--btn-text-color);
+  --btn-secondary-border: #9ba7af;
+  --btn-text-color: var(--white);
+
+  // Table
+  --even-row-color: #fbfbfb;
+  --bigtable-header-color: #f0f0f0;
+  --project-status-border-bottom-color: #ddd;
+  --table-parameters-bg-color--hover: #f0f0f0;
+  --table-stripped-bg-color--hover: #e8e8e8;
+
+  // Link
+  --link-color: #204a87;
+  --link-visited-color: #5c3566;
+  --link-dark-color: var(--black);
+  --link-dark-color--hover: var(--primary);
+  --link-dark-color--active: var(--primary-active);
+
+  // Pane
+  --pane-header-color: #eee;
+  --pane-header-border-color: #f3f3f3;
+  --pane-border-color: #f0f0f0;
+  --pane-text-color: #3b3b3b;
+  --pane-link-color: black;
+  --pane-link-color--visited: black;
+
+  // TabBar
+  --tab-bar-border-color: #f0f0f0;
+  --tab-bar-text-color: #999;
+  --tab-bar-bg--checked: #eee;
+  --tab-bar-border-color--checked: #f0f0f0;
+  --tab-bar-text-color--checked: #000;
+  --tab-bar-bg-color--hover: #eee;
+  --tab-bar-add-text-color: #0460d1;
+  --tab-bar-add-bg-color--hover: #d8efff;
+  --tab-bar-add-border-color: #d8efff;
+
+  // Side panel
+  --panel-border-color: var(#eaeff2);
+  --panel-border-color--hover: var(#cecece);
```

These rules were not having effect
```suggestion
  --panel-border-color: #eaeff2;
  --panel-border-color--hover: #cecece;
```